id-repository/credential-request-generator/pom.xml

```diff
@@ -249,29 +188,28 @@
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-context</artifactId>
-			<version>5.0.6.RELEASE</version>
 		</dependency>
 		<dependency>
 			<groupId>io.projectreactor</groupId>
 			<artifactId>reactor-core</artifactId>
-			<version>3.1.6.RELEASE</version>
 		</dependency>
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-jdbc</artifactId>
-			<version>4.3.8.RELEASE</version>
 		</dependency>
 		<dependency>
 			<groupId>org.springdoc</groupId>

```

Comment:
springdoc-openapi-ui is not required if you add springdoc-openapi-starter-webmvc-ui

Reply:
Corrected.